content/docs/1.5.2/advanced-resources/rwx-workloads.md

```diff
@@ -31,22 +31,62 @@ It is necessary to meet the following requirements in order to use RWX volumes.
-
-Please refer to [#6655](https://github.com/longhorn/longhorn/issues/6655) for more information.
-
 # Creation and Usage of a RWX Volume
 
+An RWX volume must have the access mode set to `ReadWriteMany` and the "migratable" flag disabled (*parameters.migratable: `false`*).
+
 1. For dynamically provisioned Longhorn volumes, the access mode is based on the PVC's access mode.
 2. For manually created Longhorn volumes (restore, DR volume) the access mode can be specified during creation in the Longhorn UI.
 3. When creating a PV/PVC for a Longhorn volume via the UI, the access mode of the PV/PVC will be based on the volume's access mode.
 4. One can change the Longhorn volume's access mode via the UI as long as the volume is not bound to a PVC.
 5. For a Longhorn volume that gets used by a RWX PVC, the volume access mode will be changed to RWX.
 
+## Configuring Volume Mount Options
+
+An RWX volume is accessible only when mounted via NFS. By default Longhorn uses NFS version 4.1 with the `softerr` mount option, a `timeo` value of "600", and a `retrans` value of "5".  
+
+If the NFS server becomes inaccessible, requests from NFS clients are retried according to the configured `retrans` value. Longer-duration events such as power outages and factors such as network partitions cause the requests to eventually fail.  An NFS error (`ETIMEDOUT` for the `softerr` mount option) is returned to the calling application and data loss may occur. If `softerr` is not supported, Longhorn automatically uses the `soft` mount option instead, which returns an `EIO` as the error.
+
+You can use specific mount options for new volumes. First, create a customized StorageClass with an `nfsOptions` parameter, and then create PVCs for RWX volumes using that specific StorageClass.
+
+Example:
+
+  ```yaml
+  kind: StorageClass
+  apiVersion: storage.k8s.io/v1
+  metadata:
+    name: longhorn-test
+  provisioner: driver.longhorn.io
+  allowVolumeExpansion: true
+  reclaimPolicy: Delete
+  volumeBindingMode: Immediate
+  parameters:
+    numberOfReplicas: "3"
+    staleReplicaTimeout: "2880"
+    fromBackup: ""
+    fsType: "ext4"
+    nfsOptions: "vers=4.1,noresvport,softerr,timeo=600,retrans=5"
+  ```
+
+> **Important:**
+> To create PVCs for RWX volumes using the sample StorageClass, replace the `nfsOptions` string with a customized comma-separated list of legal options.
+
+### Notes
+
+1. You must provide the complete set of options.
```

Comment:
Are there options mandatory?

Reply:
Any options not supplied will use the NFS server-side defaults, not Longhorn's own.  I should make that more explicit.
In fact, specifying nfsOptions="" should just use all the server-side defaults, whatever they are.  (I should check this.)